next.config.mjs

```diff
@@ -18,4 +18,10 @@ export default withMDX({
   images: {
     domains: ["raw.githubusercontent.com", "numpy.org", "dask.org", "chainer.org", ],
   },
+  i18n: {
+    // These are all the locales you want to support in
+    // your application
+    locales: ["en", "es", "pt"],
```

Comment:
could we add french here?

Reply:
Yes we could, but at the moment we do no have translations available for french.

Reply:
thank you for the clarification

Reply:
You can see here the state of translations 

https://scientific-python-translations.github.io/status/#xarray

We will be trying to gather more people to help with this effort :)